### PR TITLE
Remove quiet kernel option before booting

### PR DESCRIPTION
**Pull request by asdil12**
Send kernel boot messages to serial port

### CODE REVIEW

File: tests/boot/boot_to_desktop.pm [hunk 1]
```diff
@@ -14,9 +14,34 @@ use testapi;
...
+
+    save_screenshot;
+
+    # boot
+    send_key "f10";
+
     # we have some tests that waits for dvd boot menu timeout and boot from hdd
     # - the timeout here must cover it
-    wait_boot bootloader_time => 80;
+    wait_boot bootloader_time => 1;
```
Comment (sysrich) [inline tests/boot/boot_to_desktop.pm]: Why are we reducing the bootloader time from 80 to 1 here?

**Comment by coolo** [comment]
Why?

File: tests/boot/boot_to_desktop.pm (hunk 1 above)
Comment (asdil12) [inline tests/boot/boot_to_desktop.pm]: Because the bootloader isn't running anymore at this point of time

**Comment by asdil12** [comment]
To have the same log output as for normal tests

**Comment by coolo** [comment]
then remove that boot option when creating the asset and don't waste time fiddling with the boot config every time we boot

**Comment by asdil12** [comment]
done.

**Comment by coolo** [comment]
this will be restored once you install a kernel. /etc/sysconfig/bootloader is the minimum you want to patch

**Comment by asdil12** [comment]
Now modifying `/etc/default/grub`

File: tests/installation/grub_set_bootargs.pm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    select_console('root-console');
+    script_run "source /etc/default/grub";
+    script_run 'new_cmdline=`echo $GRUB_CMDLINE_LINUX_DEFAULT | sed \'s/\(^\| \)quiet\($\| \)/ /\'`';
+    script_run 'sed -i "s#GRUB_CMDLINE_LINUX_DEFAULT=\"[^\"]*\"#GRUB_CMDLINE_LINUX_DEFAULT=\"$new_cmdline\"#" /etc/default/grub';
+    script_run 'grub2-mkconfig -o /boot/grub2/grub.cfg';
+    select_console('x11');
```
Comment (coolo) [inline tests/installation/grub_set_bootargs.pm]: how can you know it's the proper console? x11 doesn't exist in all the textmode installations ?

Comment (asdil12) [inline tests/installation/grub_set_bootargs.pm]: how can I know, if it is not?

Comment (coolo) [inline tests/installation/grub_set_bootargs.pm]: if it's not existing, select_console will crash

**Comment by coolo** [comment]
I suggest to do this before shutdown/ - we only need this patching in case of a published hdd. And in there you don't really need to prep the next console - shutdown will select root-console itself

**Comment by asdil12** [comment]
Now running the test directly before `shutdown/`

**Comment by coolo** [comment]
I think we can leave it like that - *but* please move the .pm into shutdown/ too and add it to sle/main.pm

**Comment by asdil12** [comment]
done

**Comment by coolo** [comment]
let's see where it explodes first

**Pull request merged**